integration/docker-compose.pxe.yml

```diff
@@ -30,39 +14,31 @@ services:
     cap_add:
       - CAP_NET_BIND_SERVICE
       - CAP_NET_RAW
-    network_mode: service:tftp
+    networks:
+      xpu-cpu:
+        ipv4_address: 10.127.127.3
```

Comment:
Did you update the diagram to reflect any IP address changes?

Reply:
no need to update, checked. we used the same IP for TFTP and DHCP